front-end-master/landing_page/index.html

```diff
@@ -237,6 +246,10 @@ <h1 class="headline">Our Associates</h1>
     <div class="go-top">
 
     </div>
+
+    <div id="copyright">
```

Comment:
I'd rather have class names dictate css rather than id. This way, we can have the same class name used else where if we want the same css style where as you should not have the same id used more than once. So I suggest changing 'copyright' from an ID to a class name. 

Id is ok to use when you know that element will be truly unique and in cases of section reference.